### ISSUE TITLE
Decide on Elastic Beanstalk auto scaling settings for production Via 3

### ISSUE THREAD

**Issue by seanh** [hypothesis/via#14]
Before we can [Release Via 3 for production LMS users](https://github.com/hypothesis/py-proxy/issues/11) we need to set its Elastic Beanstalk autoscaling settings appropriately:

- [x] What should the minimum number of instances be? (It should be at least 2. We may want a higher minimum.)
- [x] What should the maximum number of instances be?
- [x] What should the scaling triggers be?

We should involve @robertknight and @dmfine in a discussion about this, to decide on good starting values for production Via 3.

**Comment by seanh** [comment]
In a [Slack post](https://hypothes-is.slack.com/archives/C4K6M7P5E/p1578969771098200) @hmstepanek has suggested running Via 3 with four app servers as a starting point. So we might set 4 as the autoscaling minimum. We would still need to decide:

* The autoscaling maximum
* The autoscaling triggers
* [The EC2 instance type, number of Gunicorn workers, and number of Nginx workers](https://github.com/hypothesis/py-proxy/issues/15) (separate card)

@robertknight @dmfine would be good to get your input on this

**Comment by hmstepanek** [comment]
What should the minimum number of instances be? (It should be at least 2. We may want a higher minimum.) `4`
 What should the maximum number of instances be? `12`
 What should the scaling triggers be? `sum of requests >4 increase instances, sum of requests <2 decrease instances.

**Comment by hmstepanek** [comment]
I have an [open thread on slack](https://hypothes-is.slack.com/archives/CR3E3S7K8/p1579199862035400) just to make sure I'm understanding the auto scaling policy correctly. Once that's answered, I'll make any adjustments I need to make based on the feedback and close this issue.
